src/api/app/models/project.rb

```diff
@@ -1568,6 +1568,10 @@ def missing_checks
     @missing_checks ||= calculate_missing_checks
   end
 
+  def branch?
+    name.include?(':branches:')
```

Comment:
Please also 'port' over the comment ;-) Otherwise people might start to use this for other things.